pedant/templatetags/pedant_tags.py

```diff
@@ -49,6 +49,8 @@ def ifdef(parser, token):
     block_tokens = ('elifdef', 'else', 'endifdef')
     # {% ifdef ... %}
     bits = token.split_contents()[1:]
+    if len(bits) > 1:
+        raise TemplateSyntaxError('%r is not an identifier.' % token)
```

Comment:
Without this change, the `{% ifdef %}` was failing the test that checks that you can't pass arbitrary expressions into `{% ifdef %}`. It was failing on all the versions of Django I tested this on, from 1.5 through 1.9, but I'm not sure why it was passing for you a couple years ago. I'm making the assumption that all inputs with more than one argument cannot be a single identifier, which I can't think of a counterexample for.